index.js

```diff
@@ -16,7 +17,16 @@ const getWorkflowDir = () => resolveAlfredPrefs().then(prefs => path.join(prefs,
 const readPkg = workflowDir => pathExists(workflowDir)
 	.then(exists => {
 		if (!exists) {
-			throw new Error(`Workflow directory \`${workflowDir}\` does not exist`);
+			return resolveAlfredPrefs()
+				.then(prefs => pathExists(prefs))
+				.then(exists => {
+					if (!exists) {
+						throw new Error(`Workflow directory \`${workflowDir}\` does not exist`);
```

Comment:
Since we now automatically create the workflow directory, this should say preferences directory or something instead.

Reply:
Good point 👍 I’ve changed it to “`Alfred.alfredpreferences` package” since [Alfred’s documentation](https://www.alfredapp.com/help/advanced/sync/) calls it so.